packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -1050,35 +1048,104 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
           ),
         );
 
-        // Initially the exit select button is on the left.
         final Finder exitButton = buttonFinder('EXIT SELECT MODE');
         expect(exitButton, findsOneWidget);
         final Finder moveRightButton = buttonFinder('MOVE RIGHT');
         expect(moveRightButton, findsOneWidget);
         final double initialExitButtonX = tester.getCenter(exitButton).dx;
 
-        // Move the button to the right.
         await tester.tap(moveRightButton);
         await tester.pump();
 
-        // Verify the button is now on the right.
-        expect(moveRightButton, findsNothing);
         final Finder moveLeftButton = buttonFinder('MOVE LEFT');
         expect(moveLeftButton, findsOneWidget);
-        final double exitButtonXAfterMovingRight = tester.getCenter(exitButton).dx;
-        expect(initialExitButtonX, lessThan(exitButtonXAfterMovingRight));
+        final double movedExitButtonX = tester.getCenter(exitButton).dx;
+
+        expect(initialExitButtonX, lessThan(movedExitButtonX), reason: 'LTR: should move right');
 
-        // Move the button to the left again.
         await tester.tap(moveLeftButton);
         await tester.pump();
 
-        // Verify the button is in its original position.
-        expect(moveLeftButton, findsNothing);
+        final double finalExitButtonX = tester.getCenter(exitButton).dx;
+        expect(finalExitButtonX, equals(initialExitButtonX));
+      },
+      skip: !WidgetInspectorService.instance.isWidgetCreationTracked(),
```

Comment:
Add back the comment above the `skip` (here and in test case below): `[intended] Test requires --track-widget-creation flag.` 

Skipping a test requires a justification comment.